src/pocketmine/Player.php

```diff
@@ -3302,6 +3302,7 @@ public function handleDataPacket(DataPacket $packet){
 				}
 
 				$this->inventory->remove($dropItem);
+				$this->inventory->setItemInHand(Item::get(Item::AIR, 0, 1));
```

Comment:
The item is already removed. This is possibly doing nothing and may cause further bugs.